.github/workflows/more-info-needed-closer.yml

```diff
@@ -0,0 +1,23 @@
+name: More Info Needed Closer
+on:
+  schedule:
+    - cron: 10 11 * * * # Run at 11:10 AM UTC (3:10 AM PST, 4:10 AM PDT)
+  workflow_dispatch:
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v2
+      - name: Install Actions
+        run: cd ./.github/actions && npm install --production && cd ../..
+      - name: Stale Closer
+        uses: ./.github/actions/StaleCloser
+        with:
+          labels: more info needed
+          ignoredLabels: debugger
+          closeDays: 14
```

Comment:
Is this 14 day value enough time? -- it took around 4 weeks to get enough info on the recent "No IL" regression...although in that case there happened to be a continuous stream of "activity".

Reply:
These timeouts are based on the last update, so latest activity.  The issue would need to have no activity for that number of days, (which I am increasing to 60 days, based on Bob's feedback).  It also is not automatically closed if the last commenter is not on the team (if not in additionalTeam field, does not have write or admin access to the repo, and is not a bot).